code/game/objects/items/toys.dm

```diff
@@ -816,8 +816,8 @@
 		qdel(src)
 		N.pickup(cardUser)
 		cardUser.put_in_hands(N)
-		to_chat(cardUser, "<span class='notice'>You also take [currenthand[1]] and hold it.</span>")
-
+		//to_chat(cardUser, "<span class='notice'>You also take [currenthand[1]] and hold it.</span>")
```

Comment:
Dont need this comment anymore

Reply:
is it not good practice to keep things commented out instead of deleting them, for future reference?